edu.cuny.hunter.log.core/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: Core
 Bundle-SymbolicName: edu.cuny.hunter.log.core;singleton:=true
-Bundle-Version: 1.0.0.qualifier
+Bundle-Version: 1.1.0.qualifier
```

Comment:
Is this the only plug-in that had changes?

Reply:
I bumped up the versions for log-core, log-ui, mylin-core & log-feature